docs/components/Disclosure/Web.stories.tsx

```diff
@@ -35,7 +35,7 @@ CustomTitle.args = {
   title: (
     <Flex template={["shrink", "grow"]} gap="small">
       <Icon name="sparkles" />
-      <Heading level={5} element="h4">
+      <Heading level={5} element="span">
```

Comment:
I think it's alright for us to leave it as a h4 no?

Reply:
It should be fine. However, I changed it based on that accessibility concern you mentioned above (just to gently "nudge" consumers to use "span" or any other phrasing content). I can revert this change if that would be your preference! Just LMK.

Reply:
that's fair. if we mention the details about the content type + accessibility in our docs, I'm fine with offering an example of something more accessible via the `span`.